Work Section and About Myself template one Figma

jovasal opened the pull request:
This Project contains the following activities:

- Linters.
- GitHub Flow.
- README.
- Pull Request.
- Work in a professional Way.
- Work Section to Figma
- About Myself to figma

jovasal committed:
ok

ok

jovasal committed:
change

change

jovasal committed:
k

k

jovasal committed:
ok

ok

jovasal committed:
ok

ok

jovasal committed:
ok

ok

jovasal committed:
ok

ok

jovasal committed:
ok

ok

jovasal committed:
k

k

jovasal committed:
k

k

jovasal committed:
i

i

jovasal committed:
l

l

jovasal committed:
h

h

jovasal committed:
l

l

jovasal committed:
K

K

jovasal committed:
done

done

jovasal committed:
k

k

jovasal committed:
4

4

jovasal committed:
l

l

jovasal committed:
k

k

jovasal committed:
done

done

divinecharlotte commented on Index.html:
```diff
@@ -0,0 +1,322 @@
+          </ul>
+        </div>
+      </section>
+      <main class="secpa">
```
- [x] kindly consider displaying all the cards in this section in the center so that the grey background is visible on the left side too to meet the design in the Figma check the images below for more clarification 👇 
- your design 👇 
<img width="325" alt="Screen Shot 2023-06-03 at 07 16 57" src="https://github.com/jovasal/Finish-Mob-Figma/assets/60146030/4f546b5c-bb59-4a24-9ba7-2a49b7eaaaa8">


- Figma design 👇 
<img width="354" alt="Screen Shot 2023-06-03 at 07 15 35" src="https://github.com/jovasal/Finish-Mob-Figma/assets/60146030/700b275d-7fb6-45df-b3e3-9a361f65be44">

jovasal replied:
Thank you so much!!

HTML CSS JAVASCRIPT is ok now, Also All project are in center:

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/e67e4065-f677-4408-8014-ec7e4f9031fe)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/c24bc7ad-2fc0-48cb-b98b-055ddf744ef2)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/c7e1ac52-2d64-415d-a523-54bc936d032a)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/8d0aab92-c301-4bff-af27-71cd56cf24cc)

All comments were fixed successfully.

divinecharlotte commented on Index.html:
```diff
@@ -0,0 +1,322 @@
+                    sign-ups required.
+                  </p>
+                </div>
+                <div class="botonic">
+                  <h3 class="tonichtml">html</h3>
+                  <h3 class="toniccss">css</h3>
+                  <h3 class="tonicJava">JavaScript</h3>
+                </div>
```
- [x] kindly consider displaying this list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li` as the are list elements and remember its for semantic reasons

jovasal replied:
Changes were made succesfully.

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/745c947e-248b-41d6-a7a2-d11cba901e15)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/fad0f0bd-a133-443f-a96b-e491debccc20)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/2b53697d-8194-443f-ae92-dccf35ddd50d)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/0b067bad-518a-4879-b614-00d3700965bd)

jovasal committed:
Update README.md

divinecharlotte commented on Index.html:
```diff
@@ -0,0 +1,322 @@
+          </ul>
+        </nav>
+      </main>
+      <section class="contact-me">
```
- [x] kindly consider giving this section a border right radius as designed in Figma  after doing that consider also this:
- [x] 1. adding the technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li`
1. adding  technology in each category(e.g., "JavasScript", "HTML" and "CSS")  also a list `ul` and each of them a `li`  and also consider designing the technology in each category(e.g., "JavasScript", "HTML" and "CSS") as designed on Figma
 to make your design  an exact duplicate of the given design and lists for semantic reasons check the images below for more details 
- your design 👇 
<img width="188" alt="Screen Shot 2023-06-03 at 07 23 09" src="https://github.com/jovasal/Finish-Mob-Figma/assets/60146030/cd8a6949-d630-4437-9f12-419ca5d62bd0">

- Figma design 👇 
<img width="260" alt="Screen Shot 2023-06-03 at 07 29 54" src="https://github.com/jovasal/Finish-Mob-Figma/assets/60146030/6b8a9ceb-793e-4197-afa5-8d2f0c61aeb0">

jovasal replied:
Border radius were made. Background color is too similar and is not easy to see it. I made a change just for check this point and return it.

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/e681d855-dfe4-49fb-ab6a-16a5fe959786)

jovasal replied:
Language is ok now as described Figma:
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/192dbbd3-8572-431f-92c3-3041bfe14386)

jovasal replied:
Technologies were used as described in recommendations. Also Code were fixed and ordered.

divinecharlotte reviewed (changes requested):
Hi @jovasal         , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- descriptive readme 👍 
- Linters are passing 👍 
 




 
![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)
## Required Changes ♻️





_Check the comments under the review._
## Optional suggestions

- 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jovasal committed:
Changes were made succesfully

great work

jovasal committed:
done

orginizing code

jovasal committed:
code fixed

fixted all activities

jovasal committed:
HTML ordered

HTML ordered and Languages changed

franclobo commented on Index.html:
```diff
@@ -0,0 +1,313 @@
+        </ul>
+      </div>
+    </section>
+    <main class="secpa">
```
- [x] As requested by the previous CR, please 🙏🏽 correct the margins to center the content of the works section to be similar to the Figma design 👇🏽 

https://github.com/jovasal/Finish-Mob-Figma/assets/58642949/bede7af1-9871-4ef7-b3ac-a510f9f4b72d

jovasal replied:
Hi @franclobo I understand. 
Margin we fixed and actually are running well:

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/081bb59f-b4ab-4070-88b0-b4144db7aef0)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/85e156c2-c61a-490e-af0b-c2b304326259)
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/d8ed9095-d062-415a-9b60-7dfe89c86aef)

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/2a2c1501-d884-4346-85b4-56617a1c95d5)

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/3d2a2c89-a0b7-4b23-871d-3af17a972a75)

franclobo commented on style.css:
```diff
@@ -0,0 +1,1651 @@
+  left: 375px;
+  top: 4769px;
+  background-color: #fff;
+  border-top-right-radius: 80px;
```
- [x] It seems there is something else behind the About section, please remove it to be able to see the upright corner (I changed the color to prove it)

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/58642949/1b876b1a-4265-439f-a3f5-f9cdd8ba2791)

jovasal replied:
ok. Perfect.

The color was made under the requirements described before.

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/f72a8498-bdb6-480a-96bb-91aeb629dfb0)

franclobo commented on Index.html:
```diff
@@ -0,0 +1,313 @@
+                  <li class="JavaScript1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-1.png" alt="" />
+                      <p>JavaSript</p>
+                    </div>
+                  </li>
+                  <li class="HTML1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-2.png" alt="" />
+                      <p>HTML</p>
+                    </div>
+                  </li>
+                  <li class="CSS1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-3.png" alt="" />
+                      <p>CSS</p>
```
- [ ] Please 🙏🏽 use the correct font styles to be an exact duplicate of the Figma template 👇🏽 

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/58642949/8f70746d-95ea-4f70-821b-bdc693b5560e)

jovasal replied:
I tried to use only a font family Poppins, but is not working. Each time that I tried to use it alone, Linter errors show:
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/6005c467-a8ba-4f0b-b6e6-a6e9771d3bfc)
and this error comes to:
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/d6ba6ce4-c20c-44ed-93b4-b3fe1e5b0035)
for this reason, the code is using (font-family:  "Poppins", sans-serif;) that is too similar:
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/f56949db-154e-4358-9408-9693855ea1c4)

On the other hand, Code contains the exact information in Figma template as you can see in the following image:
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/8aafad14-ca21-4ba9-aa65-2f86ca1b0d95)

franclobo commented on style.css:
```diff
@@ -0,0 +1,1651 @@
+  list-style-type: none;
+}
+
+.event18 {
+  box-sizing: border-box;
+  width: 327px;
+  height: 72px;
+  padding: 0 24px;
+  margin-left: -16px;
+  display: flex;
+  align-items: initial;
+  gap: 80px;
+  margin-top: -10px;
+  border-bottom: 1px solid rgb(223, 225, 230);
+}
```
- **[OPTIONAL]** Kindly add the right narrows to be similar to the Figma template 👇🏽 

 
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/58642949/cdd9fd9b-41f7-4d54-9461-56742d654efd)

jovasal replied:
I forgot to copy the image for that.

Actually is working fine:

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/a26e3d07-f3c1-4687-b9cb-bf0583ea17a5)

franclobo reviewed (changes requested):
Hi @jovasal, 

# ♻️ Required Changes ♻️

![soClose](https://github.com/jovasal/Finish-Mob-Figma/assets/58642949/bdc472c4-dbb4-4366-9a3d-eda97819abc9)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Good practices of HTML & CSS 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jovasal committed:
changes required

Changes requiered

jovasal committed:
changes 2PR

Changed 4 errors and linter errors that appeared after changes in Poppins family font

jovasal committed:
family font modified

font-family: "Poppins", sans-serif;

rica213 commented on Index.html:
```diff
@@ -0,0 +1,313 @@
+    <section class="contact-me">
+      <div class="head-section-co">
+        <div id="head-container-co">
+          <h2 class="head-co">
+            About <br />
+            Myself
+          </h2>
+          <p class="description-co">
+            Hello I’m a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h2 class="connect-co">LET’S CONNECT</h2>
+          <ul class="social-media-icons">
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/github.png" alt="git" />
```
- [ ] [OPTIONAL] This is an **OPTIONAL** changes, however, I **HIGHLY** recommend you to implement it because it would enhance the quality of your code 🤗 
Please add more horizontal spacing (**padding**) in this section. If you compare your design to the Figma template, you will notice discrepancy 🤖 
Kindly fix that to make your webpage beautiful 💅🏽 ❤️

rica213 reviewed (approved):
# 🟢 STATUS: APPROVED
Hi John
My name is Larissa. I am your code reviewer 🥰

You have implemented all the required changes asked by previous reviewers 😻 Thank you 💟 
I do love how you respond in the comment about your updates, nice touch 😁 😁 
Kindly check the comment section because I asked an **OPTIONAL** change ✍🏽 
Your project is complete!  It's time to merge it 🛸 🛸 
Congratulations! 🎉 

## FRIENDLY SUGGESTION
- [ ] Please write more descriptive commit messages in the future. It is essential to have a descriptive commit message because it provides a historical narrative of a project's development 💯 
You can find here, [how to write good commit messages](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/)

<img src="https://media0.giphy.com/media/py2UYwTIX5SXm/giphy.gif"/>

## Highlights

- [ ] No Linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
